### DISCUSSION TITLE
Can the component be used for the development of commercial desktop software

### DISCUSSION THREAD

**Discussion by NicFTK** [dream-num/univer#1445] [Q&A | 常见问题] [answered]
Can the component be used for the development of commercial desktop software, such as Tauri or Electron?

**Comment by jikkai** [comment, marked as answer]
Certainly, as long as you adhere to the specifications of the [Apache 2.0 license](https://github.com/dream-num/univer/blob/dev/LICENSE), you may use Univer for the development of commercial desktop software, including applications built with Tauri or Electron.
